src/Admin/UnregistrationAdmin.php

```diff
@@ -86,7 +87,10 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                 'show_filter' => true,
                 'field_type' => ChoiceType::class,
                 'field_options' => [
-                    'choices' => array_flip($this->unregistrationReasons),
+                    'choices' => array_flip(
```

Comment:
I am not sure but I don't think this would work as we would have integer in choices after `array_flip`. Let me know if I am wrong. I just like to be sure.

Reply:
Nice catch! thx :+1: